src/ace/ari_text/parsemod.py

```diff
@@ -97,11 +99,9 @@ def p_typedlit_tbl_empty(p):
 
 def p_typedlit_tbl_rows(p):
     '''typedlit : SLASH TBL structlist rowlist'''
+
+    ncol = p[3].get('c', 0).value
     nrow = len(p[4])
-    try:
-        ncol = int(p[3]['c'].value)
```

Comment:
Was this removed for a reason?
The error message for the case now will be, I think, confusing because it will fail taking the "value" attribute of a built-in int "0".